### ISSUE TITLE
Migrate kube-controller-manager to use v1beta1 Events

### ISSUE THREAD

**Issue by jfbai** [kubernetes/kubernetes#82827]
Now that new Events API is ready, we can migrate the kube-controller-manager to emit events against the new API.

This is part of #76674

**Comment by jfbai** [comment]
cc @wojtek-t @shoucongc @yastij

**Comment by jfbai** [comment]
@liggitt I'm working on migrating controllers to use v1beta1 Events, and most have already done most of the controllers, is it ok to submit one PR for all controller codes? or separate PRs for different controllers?

**Comment by liggitt** [comment]
if the changes are mechanical and easy to review, a single PR is fine

**Comment by liggitt** [comment]
/hold
for https://github.com/kubernetes/kubernetes/issues/76674#issuecomment-534101819

**Comment by alculquicondor** [comment]
/sig apps

Now that the new events API is v1, any other considerations before we can do the migration?
